components/dashboard/filters/OrderFilter.tsx

```diff
@@ -49,7 +49,7 @@ function OrderFilter({ onChange, value }: { onChange: (value: OrderFilterValue)
   const Icon = value === 'CREATED_AT,DESC' ? ArrowDownNarrowWide : ArrowDownWideNarrow;
   return (
     <Select onValueChange={value => onChange(value as OrderFilterValue)} value={value}>
-      <Button size="sm" variant="outline" asChild className="max-w-[160px] rounded-full">
+      <Button size="sm" variant="outline" asChild className="w-fit rounded-full">
```

Comment:
Nice fix to use "fit-content" here instead of a fixed value 👍 . 

However, we should set it as the max-width, since if the language string is really long and you are on a small screen we'd want the layout to limit the width instead.

```suggestion
      <Button size="sm" variant="outline" asChild className="max-w-fit rounded-full">
```